src/app/pages/LiveRadioPage/LiveRadioPage.jsx

```diff
@@ -22,6 +24,12 @@ const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${pr
 
 const audioPlaceholderImageSrc = `${staticAssetsPath}images/amp_audio_placeholder.png`;
 
+const StyledGelPageGrid = styled(GelPageGrid)`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: 100%;
```

Comment:
does this not take care of this? https://github.com/bbc/simorgh/blob/cf1310650a137063d07ff410d5e207063bf81331/src/app/components/Grid/index.jsx#L73

Reply:
Ah yes, I think it does - nice one. 

We added this because we had an issue where this `width: 100%;` wasn't being set unless wrapped in a media query - it was inheriting `width: initial', which would make the player really narrow unless the text summary filled to the max width of the container.